.github/workflows/deploy-frontend-to-preview-env.yml

```diff
@@ -88,42 +88,33 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Google Auth
-        id: auth
+      - name: Google login
         uses: google-github-actions/auth@v0
         with:
           credentials_json: ${{ secrets.GCP_SA_CREDENTIALS }}
 
-      - name: Setup Cloud SDK
+      - name: Setup Cloud SDK(gcloud)
         uses: google-github-actions/setup-gcloud@v0
 
-      - name: Login to GCR
-        id: docker-auth
-        uses: docker/login-action@v1
-        with:
-          registry: ${{ secrets.REGISTRY_URL }}
-          username: _json_key
-          password: ${{ secrets.GCP_SA_CREDENTIALS }}
+      - name: Create .env file
+        run: gcloud secrets versions access latest --secret="sta-env-netmanager" > netmanager/.env
 
-      - name: NPM Setup and Build
+      - name: npm Install and Build
         uses: actions/setup-node@v2
         with:
           node-version: "12"
       - run: |
           cd netmanager/
-
-          export REACT_APP_BASE_URL=${{ secrets.REACT_APP_BASE_URL_STAGE }}
```

Comment:
These values are now being picked directly from secret manager and added to a .env file